single_kernel_mongo/managers/backups.py

```diff
@@ -141,6 +153,92 @@ def backup_id(self) -> str:
     def backup_id(self, value: str):
         self._backup_id = value
 
+    def _get_bucket_resource(self, credentials: dict[str, str]) -> Bucket:
+        """Get the Bucket resource from the s3 connection.
+
+        Returns:
+            Bucket: the s3 bucket for uploading/downloading backups
+        """
+        s3_resource = boto3.resource(
+            "s3",
+            region_name=credentials.get("region"),
+            endpoint_url=credentials["endpoint"],
+            aws_access_key_id=credentials["access-key"],
+            aws_secret_access_key=credentials["secret-key"],
+            config=BotoConfig(
+                # https://github.com/boto/boto3/issues/4400#issuecomment-2600742103
+                request_checksum_calculation="when_required",
+                response_checksum_validation="when_required",
+            ),
+            verify=(TRUST_STORE_PATH / TrustStoreFiles.PBM.value)
```

Comment:
Could we replace the value of the `TRUST_STORE_PATH` with a localized path? (withing `SNAP_DATA`)

Reply:
for k8s it should be fine, however the path on VM is still `/usr/local/share/ca-certificates` and not relative to `SNAP_DATA`

Reply:
Yes but we have not decided on an implementation for those paths and this addition would imply ours of development and new snap + rock releases to add the correct environment variables.
When I say "On VM that path is already used" it means that this is the path that we're currently using when storing that exact certificate for PBM anyway.
I had asked if there would be a need for such a spec/implementation for certificates in a Data Office Hour and the answer seemed to be that there was no need for that from mykola.